server/pairing/payload_receiver.go

```diff
@@ -94,7 +99,8 @@ func NewAccountPayloadReceiver(e *PayloadEncryptor, p *AccountPayload, config *R
 
 	return NewBaseBasePayloadReceiver(e, NewPairingPayloadMarshaller(p, l), aps,
```

Comment:
would you mind also correct this name to `NewBasePayloadReceiver` ? thank you @igor-sirotin